### ISSUE TITLE
Indicate project workingness somehow

### ISSUE THREAD

**Issue by carolynvs** [cnabio/cnab.io#30]
On the community page we'd like to let people know about projects and libraries not only so that people can use them, but so that people looking to contribute find each other instead of starting new splinter projects.

That may mean that some projects aren't as up-to-date with our rapidly changing spec as the cnab-go library for example. So it would be great to either come up with a quick way to let people know that it's working with the latest spec on the community projects page and also encourage projects to let people know on their readme's if they work with CNAB Spec v1.x so that people don't lose any time trying to figure that out themselves.

What would be really cool is to figure out how to make a badge that people could just link to that says "CNAB 1.1" and it would return the image.

Here's my suggestion. What do you think?

---

## Go
* cnab-go `CNAB 1.0.1` ✅ 

## .NET
* CNAB.NET `CNAB 1.0.0-WD` ⚠️ 

**Key**
✅ - The referenced CNAB spec version is completely supported.
⚠️ - The referened CNAB spec version is targeted but not fully supported

**Comment by vdice** [comment]
Good call; this would be helpful/relevant info.  Perhaps for a first go 'round, we can manually add the CNAB spec version info, as illustrated above, after reviewing each project.

> What would be really cool is to figure out how to make a badge that people could just link to that says "CNAB 1.1" and it would return the image.

This would be!  It may necessitate the development of a canonical validation authority that would process a runtime and determine which spec version the runtime implements.  We've seen a need for such validation (and have multiple tickets in relevant repos) and this would be another use case/motivation for it: consuming the results of validation, display via a badge, etc.
